pkg/rpcclient/wsclient.go

```diff
@@ -929,6 +979,25 @@ func (c *WSClient) ReceiveNotaryRequests(flt *neorpc.NotaryRequestFilter, rcvr c
 	return c.performSubscription(params, r)
 }
 
+// ReceiveMempoolEvents registers provided channel as a receiver for new mempool
+// events. Events can be filtered by the given MempoolEventFilter, nil value doesn't
+// add any filter. See WSClient comments for generic Receive* behaviour details.
```

Comment:
Add a note about filter fields designation, refer to the documentation of ReceiveNotaryRequests as an example.